### DISCUSSION TITLE
ROR CURVE/BEAN CURVE MISSING

### DISCUSSION THREAD

**Discussion by CoffeeByDesign** [artisan-roaster-scope/artisan#1261] [General]
I am using software on both a s35/s70 losing. On the 70 I lost my bean curve and on the 35 I lost my ROR curve. Any suggestions on how to get those back? I'm also having a hard time downloading the latest software. I think I'm one behind the latest update.

**Comment by MAKOMO** [comment]
> I'm also having a hard time downloading the latest software. I think I'm one behind the latest update.

It is always advisable to upgrade to the latest Artisan version which is linked from the Artisan homepage https://artisan-scope.org/ under the button "Install Now". This link leads to the latest release on GitHub which is v2.8.4 of the time of writing. Check the linked Release Notes and the Supported Platforms first. Then scroll down a bit to find the downloads 
 for all supported platform.

**Comment by CoffeeByDesign** [comment]
Does anyone know how to get a ROR curve to show up? It seemed to disappear and I cannot figure out how to get it back on. 

Best

**Reply by MAKOMO** [reply]
Lots of lost curves lately. Note that the RoR curve is only shown after the CHARGE event has be set. Ensure that you have the RoR Curves enabled under Config >> Curves, 1st tab RoR.

**Comment by CoffeeByDesign** [comment]
Ugh. Yeah it’s frustrating when things just disappear. Do you happen to know why my ROR curve would be a straight line? Do you have a bug fix coming for the missing lines?

From: Marko Luther ***@***.***>
Date: Wednesday, September 20, 2023 at 10:46 AM
To: artisan-roaster-scope/artisan ***@***.***>
Cc: Anthony Burrows ***@***.***>, Author ***@***.***>
Subject: Re: [artisan-roaster-scope/artisan] ROR CURVE/BEAN CURVE MISSING (Discussion #1261)

Lots of lost curves lately. Note that the RoR curve is only shown after the CHARGE event has be set. Ensure that you have the RoR Curves enabled under Config >> Curves, 1st tab RoR.

**Reply by MAKOMO** [reply]
> Do you happen to know why my ROR curve would be a straight line?

If the underlying temperature curve like the BT is raising by a constant amount the computed (BT) RoR being constant. If that temperature is constant the computed (BT) RoR will be 0.

> Do you have a bug fix coming for the missing lines?
There is no known bug on missing lines thus there is no need for a bug fix.

If you click on an LCD the corresponding lines visibility will be toggle. Maybe this is what happens here?

**Comment by CoffeeByDesign** [comment]
Nope. Not a clue. I just did what you mentioned about the ROR in the settings, and it did nothing. Everything is working except the ROR.

Understood on the bug issue.

Let me get back to you.
Best



From: Marko Luther ***@***.***>
Date: Wednesday, September 20, 2023 at 12:23 PM
To: artisan-roaster-scope/artisan ***@***.***>
Cc: Anthony Burrows ***@***.***>, Author ***@***.***>
Subject: Re: [artisan-roaster-scope/artisan] ROR CURVE/BEAN CURVE MISSING (Discussion #1261)

Do you happen to know why my ROR curve would be a straight line?

If the underlying temperature curve like the BT is neither raising nor falling this will be indicated by the computed (BT) RoR being a straight line.

Do you have a bug fix coming for the missing lines?
There is no known bug on missing lines thus there is no need for a bug fix.

If you click on an LCD the corresponding lines visibility will be toggle. Maybe this is what happens here?

**Comment by CoffeeByDesign** [comment]
Hey there,

It’s definitely not the LCDs


From: Marko Luther ***@***.***>
Date: Wednesday, September 20, 2023 at 12:23 PM
To: artisan-roaster-scope/artisan ***@***.***>
Cc: Anthony Burrows ***@***.***>, Author ***@***.***>
Subject: Re: [artisan-roaster-scope/artisan] ROR CURVE/BEAN CURVE MISSING (Discussion #1261)

Do you happen to know why my ROR curve would be a straight line?

If the underlying temperature curve like the BT is neither raising nor falling this will be indicated by the computed (BT) RoR being a straight line.

Do you have a bug fix coming for the missing lines?
There is no known bug on missing lines thus there is no need for a bug fix.

If you click on an LCD the corresponding lines visibility will be toggle. Maybe this is what happens here?

**Reply by MAKOMO** [reply]
Maybe the RoR curve is drawn, just outside of the window. Do you see RoR readings in the corresponding LCDs and are they within the limits of the right (RoR) axis?

All hard to see from here....

**Comment by CoffeeByDesign** [comment]
I do not. The LCD readings are right they just aren’t showing up while I’m roasting.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by MAKOMO** [comment]
Please go to menu Help >> Save Settings and store your Artisan settings into a file artisan-settings.aset on your computer. Rename that to artisan-settings.txt (ending in .txt). Go to GitHub and drag and drop that file to a discussion field.

I will take a look at your settings.
M

**Comment by CoffeeByDesign** [comment]
Hey there,

It won’t let me upload. Maybe you can see it from here.

From: Marko Luther ***@***.***>
Date: Wednesday, September 20, 2023 at 2:00 PM
To: artisan-roaster-scope/artisan ***@***.***>
Cc: Anthony Burrows ***@***.***>, Author ***@***.***>
Subject: Re: [artisan-roaster-scope/artisan] ROR CURVE/BEAN CURVE MISSING (Discussion #1261)
Please go to menu Help >> Save Settings and store your Artisan settings into a file artisan-settings.aset on your computer. Rename that to artisan-settings.txt (ending in .txt). Go to GitHub and drag and drop that file to a discussion field.

I will take a look at your settings.
M

**Reply by MAKOMO** [reply]
> It won’t let me upload. Maybe you can see it from here.

No. Attachments send by email are removed. The only way is to come [here](https://github.com/artisan-roaster-scope/artisan/discussions/1261#discussioncomment-7061608) to GitHub in your WebBrowser and drag-and-drop the file to end in `.txt` to comment box on this GitHub Web page.

**Comment by CoffeeByDesign** [comment]
I ended up getting it to work!

Thanks for your help.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by MAKOMO** [reply]
Could you please tell us how, such that others can learn from your discoveries?
